js/message_test.js

```diff
@@ -1049,13 +1049,7 @@ describe('Message test suite', function() {
     var nested = new proto.jspb.test.Deeply.Nested.Message();
     nested.setCount(5);
     msg.setDeeplyNestedMessage(nested);
-
-    // After a serialization-deserialization round trip we should get back the
```

Comment:
You're right, I went ahead and added the binary serialization round trip back in.